### PR TITLE
ES- 852595 Revamp Chart in DocIO

### PR DESCRIPTION
**Pull request by Venkateshmuruganandam**
Hi All,
Added the subpage for working with chart.

Task Link: https://dev.azure.com/EssentialStudio/Document%20Processing%20Libraries/_workitems/edit/852595/

Regards
Venkatesh

### CODE REVIEW

File: File-Formats/DocIO/Working-with-Charts/Format-Axis.md [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+{% endtabs %}
+
+## Secondary Value Properties
+
+The following code snippet illustrates how to customize secondary value axis in chart.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+
+//MaxCross in axis
```
Comment (M-A32) [inline File-Formats/DocIO/Working-with-Charts/Format-Axis.md:186]: //MaxCross in axis.

File: File-Formats/DocIO/Working-with-Charts/Format-Axis.md [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+chart.PrimaryValueAxis.Border.LinePattern = OfficeChartLinePattern.CircleDot;
+chart.PrimaryValueAxis.Border.LineColor = Syncfusion.Drawing.Color.Blue;
+chart.PrimaryValueAxis.Border.LineWeight = OfficeChartLineWeight.Hairline;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Customization of Font
+
+The following code snippet illustrates how to customize the border of Horizontal and vertical category axis.
```
Comment (M-A32) [inline File-Formats/DocIO/Working-with-Charts/Format-Axis.md:54]: The following code sample illustrates how to customize the border of Horizontal and vertical category axis.

File: File-Formats/DocIO/Working-with-Charts/Format-Axis.md [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+
+//Maximum value in the axis
+chart.PrimaryValueAxis.MaximumValue = 14.0;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Set the Number format in Axis
+
+The following code snippet illustrates how to set the number format in the axis.
```
Comment (M-A32) [inline File-Formats/DocIO/Working-with-Charts/Format-Axis.md:101]: The following code sample illustrates how to set the number format in the axis.

File: File-Formats/DocIO/Working-with-Charts/Format-Axis.md [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+
+//Sets category labels
+chart.PrimaryCategoryAxis.CategoryLabels = chart.ChartData[2, 1, 11, 1];
+
+{% endhighlight %}
+{% endtabs %}
+
+## Manage Gridline Visibility
+
+The following code snippet illustrates how to hide or show major and minor gridlines.
```
Comment (M-A32) [inline File-Formats/DocIO/Working-with-Charts/Format-Axis.md:127]: The following code sample illustrates how to hide or show major and minor gridlines.

File: File-Formats/DocIO/Working-with-Charts/Format-Axis.md [hunk 5]
```diff
@@ -0,0 +1,218 @@
...
+
+//Set title for secondary value axis
+chart.SecondaryValueAxis.Title = "Temperature,deg.F";
+
+{% endhighlight %}
+{% endtabs %}
+
+## Secondary Value Properties
+
+The following code snippet illustrates how to customize secondary value axis in chart.
```
Comment (M-A32) [inline File-Formats/DocIO/Working-with-Charts/Format-Axis.md:181]: The following code sample illustrates how to customize secondary value axis in chart.

File: File-Formats/DocIO/Working-with-Charts/Format-Chart-Area.md [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+FileStream imageStream = new FileStream("Data/Image.png", FileMode.Open, FileAccess.Read);
+Image image = Image.FromStream(imageStream);
+chartArea.Fill.UserPicture(image, "image");
+
+{% endhighlight %}
+{% endtabs %}
+
+## Set the Transparency level
+
+The following code snippet illustrates how to make transparency in chart area.
```
Comment (M-A32) [inline File-Formats/DocIO/Working-with-Charts/Format-Chart-Area.md:61]: The following code sample illustrates how to make transparency in chart area.

File: File-Formats/DocIO/Working-with-Charts/Format-Chart-Area.md [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+
+// Set the transparency of the chart area.
+chartArea.Fill.Transparency = 0.5;
+
+{% endhighlight %}
+{% endtabs %}
+
+N> [Transparency](https://help.syncfusion.com/cr/file-formats/Syncfusion.OfficeChart.IOfficeFill.html#Syncfusion_OfficeChart_IOfficeFill_Transparency) is only applicable when [FillType](https://help.syncfusion.com/cr/file-formats/Syncfusion.OfficeChart.IOfficeFill.html#Syncfusion_OfficeChart_IOfficeFill_FillType) is set as SolidColor. Color-shaded fill is represented as a floating-point value ranging from 0.0 (Clear) to 1.0 (Opaque).
+
+The complete code snippet illustrating the above options is shown below.
```
Comment (M-A32) [inline File-Formats/DocIO/Working-with-Charts/Format-Chart-Area.md:74]: The complete code sample illustrating the above options is shown below.

File: File-Formats/DocIO/Working-with-Charts/Format-Chart-Title.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+
+// Set the chart title.
+chart.ChartTitle = "Purchase Details";
+
+{% endhighlight %}
+{% endtabs %}
+
+## Customize the Chart Title Area
+
+The following code snippet illustrates how to customize the chart title area.
```
Comment (M-A32) [inline File-Formats/DocIO/Working-with-Charts/Format-Chart-Title.md:29]: The following code sample illustrates how to customize the chart title area.

File: File-Formats/DocIO/Working-with-Charts/Format-Data-Lables.md [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+documentation: UG
+---
+
+# Data Labels
+
+Data labels play a crucial role in enhancing the comprehensibility of a chart. By displaying specific information related to a data series or individual data points, **data labels** provide valuable context. Using Syncfusion [.NET Core Word (DocIO)](https://www.syncfusion.com/document-processing/word-framework/net-core/word-library) library you can customize the **data labels** in the chart.
+
+## Enable Data Labels in Chart
+
+The following code snippet illustrates how to visible the data label in chart.
```
Comment (M-A32) [inline File-Formats/DocIO/Working-with-Charts/Format-Data-Lables.md:15]: The following code sample illustrates how to visible the data label in chart.

File: File-Formats/DocIO/Working-with-Charts/Format-Data-Lables.md [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+chart.Series[0].DataPoints.DefaultDataPoint.DataLabels.Size = 8;
+// Change the color of the data labels 
+chart.Series[0].DataPoints.DefaultDataPoint.DataLabels.Color = OfficeKnownColors.Red;
+// Make the data labels bold
+chart.Series[0].DataPoints.DefaultDataPoint.DataLabels.Bold = true;
+// Set the font name for the data labels
+chart.Series[0].DataPoints.DefaultDataPoint.DataLabels.FontName = "calibri";
+// Make the data labels italic
+chart.Series[0].DataPoints.DefaultDataPoint.DataLabels.Italic = true;
+// Set the size of the data labels
```
Comment (M-A32) [inline File-Formats/DocIO/Working-with-Charts/Format-Data-Lables.md:43]: // Set the size of the data labels.

File: File-Formats/DocIO/Working-with-Charts/Format-Data-Lables.md [hunk 11]
```diff
@@ -0,0 +1,78 @@
...
+chart.Series[0].DataPoints.DefaultDataPoint.DataLabels.Size = 3;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Set the Position of Data Labels
+
+The following code snippet illustrates how to set the position of the data label in chart.
```
Comment (M-A32) [inline File-Formats/DocIO/Working-with-Charts/Format-Data-Lables.md:51]: The following code sample illustrates how to set the position of the data label in chart.

File: File-Formats/DocIO/Working-with-Charts/Format-Data-Lables.md [hunk 12]
```diff
@@ -0,0 +1,78 @@
...
+
+// Set the position of data labels for the first series.
+chart.Series[0].DataPoints.DefaultDataPoint.DataLabels.Position = OfficeDataLabelPosition.Center;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Resize the Data Labels
+
+The following code snippet illustrates how to resize the data label in chart.
```
Comment (M-A32) [inline File-Formats/DocIO/Working-with-Charts/Format-Data-Lables.md:64]: The following code sample illustrates how to resize the data label in chart.

File: File-Formats/DocIO/Working-with-Charts/Format-Legend.md [hunk 13]
```diff
@@ -0,0 +1,150 @@
+---
+title: Modify the appearance of legend | Syncfusion
+description: Learn how to modify the appearance of legend in a chart in a  Word document using Syncfusion .NET Core Word (DocIO) library without Microsoft Word.
```
Comment (M-A32) [inline File-Formats/DocIO/Working-with-Charts/Format-Legend.md:3]: description: Learn how to modify the appearance of legend in a chart in a Word document using Syncfusion .NET Core Word (DocIO) library without Microsoft Word.

File: File-Formats/DocIO/Working-with-Charts/Format-Legend.md [hunk 14]
```diff
@@ -0,0 +1,150 @@
...
+platform: file-formats
+control: DocIO
+documentation: UG
+---
+
+# Legend
+
+Legends are visual **pictorial** hints that provide a viewer information that helps them understand an chart. Using Syncfusion [.NET Core Word (DocIO)](https://www.syncfusion.com/document-processing/word-framework/net-core/word-library) library you can customize the **legend** in the chart.
```
Comment (M-A32) [inline File-Formats/DocIO/Working-with-Charts/Format-Legend.md:11]: Legends are visual **pictorial** hints that provide a viewer information that helps them understand an chart. Using Syncfusion [.NET Core Word (DocIO)](https://www.syncfusion.com/document-processing/word-framework/net-core/word-library) library, you can customize the **legend** in the chart.

File: File-Formats/DocIO/Working-with-Charts/Format-Legend.md [hunk 15]
```diff
@@ -0,0 +1,150 @@
...
+
+## Set the Position of Legend
+
+The following code snippet illustrates how to set the legend position in chart.
```
Comment (M-A32) [inline File-Formats/DocIO/Working-with-Charts/Format-Legend.md:15]: The following code sample illustrates how to set the legend position in chart.

File: File-Formats/DocIO/Working-with-Charts/Format-Legend.md [hunk 16]
```diff
@@ -0,0 +1,150 @@
...
+
+//Legend without overlapping the chart.
+ chart.Legend.IncludeInLayout = true;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Customization of Border
+
+The following code snippet illustrates how to modify the border of the legend in chart.
```
Comment (M-A32) [inline File-Formats/DocIO/Working-with-Charts/Format-Legend.md:41]: The following code sample illustrates how to modify the border of the legend in chart.

File: File-Formats/DocIO/Working-with-Charts/Format-Legend.md [hunk 17]
```diff
@@ -0,0 +1,150 @@
...
+
+//Modifies the legend entry.
+chart.Legend.LegendEntries[0].IsDeleted = true;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Manage Legend Visibility
+
+The following code snippet illustrates how to hide the legend in chart.
```
Comment (M-A32) [inline File-Formats/DocIO/Working-with-Charts/Format-Legend.md:88]: The following code sample illustrates how to hide the legend in chart.

File: File-Formats/DocIO/Working-with-Charts/Format-Legend.md [hunk 18]
```diff
@@ -0,0 +1,150 @@
...
+{% endtabs %}
+
+## Set the Position using Layout
+
+The following code snippet illustrates how to position the legend using layout.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+
+//Manually positioning chart legend area using Layout
```
Comment (M-A32) [inline File-Formats/DocIO/Working-with-Charts/Format-Legend.md:119]: //Manually positioning chart legend area using Layout.

File: File-Formats/DocIO/Working-with-Charts/Format-Legend.md [hunk 19]
```diff
@@ -0,0 +1,150 @@
...
+chart.Legend.Layout.LeftMode = LayoutModes.edge;
+chart.Legend.Layout.TopMode = LayoutModes.edge;
+
+//Manually positioning chart legend area using Manual Layout
```
Comment (M-A32) [inline File-Formats/DocIO/Working-with-Charts/Format-Legend.md:123]: //Manually positioning chart legend area using Manual Layout.

File: File-Formats/DocIO/Working-with-Charts/Format-Legend.md [hunk 20]
```diff
@@ -0,0 +1,150 @@
...
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+
+//Manually resizing chart legend area using Layout
+chart.Legend.Layout.Left = 400;
+chart.Legend.Layout.Top = 150;
+chart.Legend.Layout.Width = 150;
+chart.Legend.Layout.Height = 100;
+
+//Manually resizing chart legend area using Manual Layout
```
Comment (M-A32) [inline File-Formats/DocIO/Working-with-Charts/Format-Legend.md:143]: //Manually resizing chart legend area using Manual Layout.

File: File-Formats/DocIO/Working-with-Charts/Format-Legend.md [hunk 21]
```diff
@@ -0,0 +1,150 @@
...
+{% endtabs %}
+
+## Resize the Legend
+
+The following code snippet illustrates how to resize the legend in chart.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+
+//Manually resizing chart legend area using Layout
```
Comment (M-A32) [inline File-Formats/DocIO/Working-with-Charts/Format-Legend.md:137]: //Manually resizing chart legend area using Layout.

File: File-Formats/DocIO/Working-with-Charts/Format-Plot-Area.md [hunk 22]
```diff
@@ -0,0 +1,127 @@
+---
+title: Modify the appearance of plot area | Syncfusion
```
Comment (M-A32) [inline File-Formats/DocIO/Working-with-Charts/Format-Plot-Area.md:2]: title: Modify the Appearance of Plot Area | Syncfusion

File: File-Formats/DocIO/Working-with-Charts/Format-Plot-Area.md [hunk 23]
```diff
@@ -0,0 +1,127 @@
...
+{% endtabs %}
+
+## Customization of Color
+
+The following code snippet shows how to fill the color in plot area.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+
+ //Sets fill type and color
```
Comment (M-A32) [inline File-Formats/DocIO/Working-with-Charts/Format-Plot-Area.md:37]: //Sets fill type and color.

File: File-Formats/DocIO/Working-with-Charts/Format-Plot-Area.md [hunk 24]
```diff
@@ -0,0 +1,127 @@
...
+chartPlotArea.Fill.Transparency = 0.5;
+
+{% endhighlight %}
+{% endtabs %}
+
+N> [Transparency](https://help.syncfusion.com/cr/file-formats/Syncfusion.OfficeChart.IOfficeFill.html#Syncfusion_OfficeChart_IOfficeFill_Transparency) is only applicable when [FillType](https://help.syncfusion.com/cr/file-formats/Syncfusion.OfficeChart.IOfficeFill.html#Syncfusion_OfficeChart_IOfficeFill_FillType) is set as SolidColor. Color-shaded fill is represented as a floating-point value ranging from 0.0 (Clear) to 1.0 (Opaque).
+
+## Set the position of Plot Area
+
+The following code snippet shows how to make transparency in plot area.
```
Comment (M-A32) [inline File-Formats/DocIO/Working-with-Charts/Format-Plot-Area.md:77]: The following code sample shows how to make transparency in plot area.

File: File-Formats/DocIO/Working-with-Charts/Format-Plot-Area.md [hunk 25]
```diff
@@ -0,0 +1,127 @@
...
+{% endhighlight %}
+{% endtabs %}
+
+The complete code snippet illustrating the above options is shown below.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+
+FileStream fileStreamPath = new FileStream("Data/Template.docx", FileMode.Open, FileAccess.Read, FileShare.ReadWrite);
+//Open an existing document from file system through constructor of WordDocument class
```
Comment (M-A32) [inline File-Formats/DocIO/Working-with-Charts/Format-Plot-Area.md:96]: //Open an existing document from file system through constructor of WordDocument class.

File: File-Formats/DocIO/Working-with-Charts/Format-Series.md [hunk 26]
```diff
@@ -0,0 +1,195 @@
...
+{% endtabs %}
+
+## Add High-Low Lines
+
+The following code snippet illustrates how to add high-low lines.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+
+//Set HasHighLowLines property to true.
```
Comment (M-A32) [inline File-Formats/DocIO/Working-with-Charts/Format-Series.md:119]: //Set the HasHighLowLines property to true.

**Review by M-A32 — CHANGES_REQUESTED** [review]
I have completed the review.

File: File-Formats/DocIO/Format-Axis.md [hunk 27]
```diff
@@ -0,0 +1,218 @@
+---
```
[the remark this answers is not in the document]
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Format-Axis.md]: modified

File: File-Formats/DocIO/Format-Axis.md [hunk 28]
```diff
@@ -0,0 +1,218 @@
...
+title: Modify the appearance of axes in chart | Syncfusion
+description: Learn how to modify the appearance of axes in a chart using Syncfusion .NET Core Word (DocIO) library without Microsoft Word or interop dependencies.
```
[the remark this answers is not in the document]
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Format-Axis.md]: Modified

File: File-Formats/DocIO/Working-with-Charts/Format-Chart-Title.md [hunk 29]
```diff
@@ -0,0 +1,201 @@
...
+chart.ChartTitleArea.Layout.Left = 10
+
+' Manually resizing chart title area using Manual Layout.
+chart.ChartTitleArea.Layout.ManualLayout.Top = 0.005
+chart.ChartTitleArea.Layout.ManualLayout.Left = 0.26
+
+{% endhighlight %}
+{% endtabs %}
+
+The complete code snippet illustrating the above options is shown below.
```
Comment (florenceliliana) [inline File-Formats/DocIO/Working-with-Charts/Format-Chart-Title.md:115]: The complete code sample illustrating the above options is shown as follows.

File: File-Formats/DocIO/Working-with-Charts/Format-Chart-Title.md [hunk 30]
```diff
@@ -0,0 +1,201 @@
...
+chart.ChartTitleArea.Color = OfficeKnownColors.Red
+chart.ChartTitleArea.Underline = OfficeUnderline.DashLong
+chart.ChartTitleArea.Size = 14
+
+{% endhighlight %}
+{% endtabs %}
+
+## Resize the Chart Title Area
+
+The following code snippet illustrates how to resize the chart title area.
```
Comment (florenceliliana) [inline File-Formats/DocIO/Working-with-Charts/Format-Chart-Title.md:77]: The following code sample illustrates how to resize the chart title area.

File: File-Formats/DocIO/Working-with-Charts/Format-Chart-Title.md [hunk 31]
```diff
@@ -0,0 +1,201 @@
...
+
+' Set the chart title.
+chart.ChartTitle = "Purchase Details"
+
+{% endhighlight %}
+{% endtabs %}
+
+## Customize the Chart Title Area
+
+The following code snippet illustrates how to customize the chart title area.
```
Comment (florenceliliana) [inline File-Formats/DocIO/Working-with-Charts/Format-Chart-Title.md:40]: The following code sample illustrates how to customize the chart title area.

File: File-Formats/DocIO/Working-with-Charts/Format-Chart-Title.md [hunk 32]
```diff
@@ -0,0 +1,201 @@
...
+documentation: UG
+---
+
+# Chart Title
+
+Chart title is a concise description at the top of a chart, offering context and clarity for the data displayed. Using Syncfusion [.NET Core Word (DocIO)](https://www.syncfusion.com/document-processing/word-framework/net-core/word-library) library, you can customize the chart title in the chart.
+
+## Set the Chart Title Name
+
+The following code snippet illustrates how to set the chart title name.
```
Comment (florenceliliana) [inline File-Formats/DocIO/Working-with-Charts/Format-Chart-Title.md:15]: The following code sample illustrates how to set the chart title name.

**Review by florenceliliana — COMMENTED** [review]
The content review is completed.

**Pull request merged**